### PR TITLE
7412: Add ordapi get

### PR DESCRIPTION
**Pull request by kat1906**
Story: [AB#7412](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7412)
Task: [AB#7507](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7507)

### CODE REVIEW

File: app/pages/sections/service-recipients/controller.js [hunk 1]
```diff
@@ -4,12 +4,34 @@ import { getEndpoint } from '../../../endpoints';
...
+      accessToken,
+      logger,
+    });
+    logger.info(`Service recipients for organisation with id: ${orgId} found in OAPI. ${serviceRecipientsData}`);
+  } catch (err) {
+    logger.error(`No selected service recipients data returned from OAPI for order id: ${orgId}. ${err}`);
+    throw new Error();
+  }
+
+  try {
```
Comment (alibombaye) [inline app/pages/sections/service-recipients/controller.js:23]: shouldn't this try be inside the one above? we don't want to do this if the other one fails

File: app/pages/sections/service-recipients/controller.js [hunk 2]
```diff
@@ -4,12 +4,34 @@ import { getEndpoint } from '../../../endpoints';
...
+    const selectedServiceRecipientsEndpoint = getEndpoint({ endpointLocator: 'getSelectedServiceRecipientsFromOrdapi', options: { orderId } });
+    selectedServiceRecipientsData = await getData({
+      endpoint: selectedServiceRecipientsEndpoint,
+      accessToken,
+      logger,
+    });
+    logger.info(`${selectedServiceRecipientsData.serviceRecipients.length} selected service recipients found in ORDAPI.`);
+  } catch (err) {
+    logger.error(`No service recipients data returned from OAPI for org id: ${orgId}. ${err}`);
```
Comment (alibombaye) [inline app/pages/sections/service-recipients/controller.js:32]: logger message needs updating?

File: app/pages/sections/service-recipients/controller.test.js [hunk 3]
```diff
@@ -19,20 +19,27 @@ describe('service-recipients controller', () => {
 
     it('calls getData once with correct params', async () => {
```
Comment (alibombaye) [inline app/pages/sections/service-recipients/controller.test.js:20]: call twice

File: app/pages/sections/service-recipients/controller.js (hunk 1 above)
Comment (kat1906) [reply app/pages/sections/service-recipients/controller.js:23]: true

Comment (kat1906) [reply app/pages/sections/service-recipients/controller.js:17]: I've removed the first try catch then - it was confusing at the time!

**Pull request merged**